pkgs/development/compilers/graalvm/community-edition/default.nix

```diff
@@ -0,0 +1,21 @@
+{ callPackage, Foundation }:
+
+let
+  mkGraal = opts: callPackage (import ./repository.nix opts) {
+    inherit Foundation;
+  };
+in {
+  inherit mkGraal;
```

Comment:
It doesn't seem this is being used. Can't it be deleted :thinking: ?

Reply:
this make it possible to any derivation use it's own graalvm version, like we do for mkFlutter, with that we can have babashka using a graalvm different from clojure-lsp for example

Reply:
Yeah, but this isn't exposed on `all-packages.nix`. So unless I misunderstood, this can't be used for this purpose.

Reply:
I'll expose it on all-packages.nix so

Reply:
@thiagokokada we expose the `graalvmCEPackages`, so it would be possible use like `graalvmCEPackages.mkGraal` right? just like the flutter one